runtime/vm.go

```diff
@@ -22,15 +22,15 @@ const (
 )
 
 var (
-	truePtr      = &objects.TrueValue
-	falsePtr     = &objects.FalseValue
-	undefinedPtr = &objects.UndefinedValue
+	truePtr      = objects.TrueValue
+	falsePtr     = objects.FalseValue
+	undefinedPtr = objects.UndefinedValue
 )
 
 // VM is a virtual machine that executes the bytecode compiled by Compiler.
 type VM struct {
 	constants      []objects.Object
-	stack          []*objects.Object
+	stack          []objects.Object
 	sp             int
 	globals        []*objects.Object
```

Comment:
this can be `[]objects.Object` now.

Reply:
also `vm.builtinModules` and `script.Variable.value` don't have to be object pointers anymore.

Reply:
if `globals` are non-pointer slice, there are some code that be can be reduced. For example, the following code in `OpSetGlobal` case:

```golang
obj := v.stack[v.sp]
v.globals[globalIndex] = obj
```

can be

```golang
v.globals[globalIndex] = v.stack[v.sp]
```

Reply:
plus, we can remove `objectPtr()` utility functions in several different places.